addons/pos_restaurant/static/src/app/tip_receipt/tip_receipt.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
 
+    <t t-name="pos_restaurant.MultipleTipReceipt">
+        <div class="pos-receipt">
+            <t t-call="pos_restaurant.TipReceipt"/>
+            <p style="page-break-after:always;"/>
+            <t t-call="pos_restaurant.TipReceipt"/>
```

Comment:
what's the idea here ? You want to have the same content twice?

Reply:
but then you have to make this work just for pdfs, not for normal receipts too; this implementation will affect normal receipts too